spec/swagger.yaml

```diff
@@ -603,12 +607,7 @@ paths:
       description: |
         This uploads original source content to Smartling. 
         
-        **Example**
-        ```bash
-          curl -X POST -H "Authorization: Bearer {token}" -F "file=@file.properties;type=text/plain" -F "fileUri=file.properties"  -F "fileType=javaProperties" 'https://api.smartling.com/files-api/v2/projects/{projectId}/file'
-        ```
-
-        The example above will upload your Java properties file directly into the Smartling project identified by the `projectId`. Smartling will ingest this file, parse out the keys and text as strings for translation. At this point, content is ready for translation. 
+        The example shown will upload your Java properties file directly into the Smartling project identified by the `projectId`. Smartling will ingest this file, parse out the keys and text as strings for translation. At this point, content is ready for translation. 
```

Comment:
i might say "The curl example provided" so they know its in curl where the specific example is

Reply:
Fixed.

Reply:
i see still one more place it is outdated. Otherwise it is good